recipes/cWB/meta.yaml

```diff
@@ -0,0 +1,99 @@
+{% set name = "cWB" %}
+{% set version = "6.4.2" %}
+
+package:
+   name: "{{ name|lower }}"
+   version: "{{ version }}"
+
+source:
+   url: https://gitlab.com/gwburst/public/library/-/archive/cWB-{{ version }}_test_conda_v2/library-cWB-{{ version }}_test_conda_v2.tar.gz
+   sha256: b5318a2aba3e029a3b4942d7a51d8878d6aa89dee88b34a5096a8de37dce7bad
+   patches:
+     # use rootcling from build env when cross-compiling
+     - rootcling-cross.patch  # [build_platform != target_platform]
+
+build:
+  error_overdepending: true
+  error_overlinking: true
+  number: 0
+  skip: true  # [win or osx]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - cmake
+    - make  # [unix]
+    - pkg-config  # [unix]
+    - root_base  # [build_platform != target_platform]
+    - sysroot_linux-64 2.17  # [linux64]
+  host:
+    - healpix_cxx >=3.80
+    - root_base
+    - libframel
+    - liblal
+    - liblalinspiral
+    - liblalburst
+    - liblalmetaio
+    - liblalsimulation
+  run:
+    - healpix_cxx >=3.80
+    - root_base
+    - root
+    - libframel
+    - liblal
+    - liblalinspiral
+    - liblalburst
+    - liblalmetaio
+    - liblalsimulation
+    - tcsh
+    - sundials
```

Comment:
So it was added because this code uses root to compile code at runtime and some part of the code links against the sundials library. But after confirming with the developer it seems that `sundials` is not used since the code that uses is not currently supported so this can be safely removed.